rebasehelper/specfile.py

```diff
@@ -170,12 +170,15 @@ def _get_initial_sources_list(self):
             archive = [x for x in Archive.get_supported_archives() if src[0].endswith(x)]
             # if the source is a remote file, download it
             if archive:
-                if remote_files_re.search(src[0]) and self.download:
+                if remote_files_re.search(src[0]) and self.download and not os.path.isfile(abs_path):
```

Comment:
From my perspective it depends on what we are searching for. If we are searching for a file then from my perspective is better to use isfile. Cause from the name we now what we are testing. 
Otherwise we should use exists. But this depends on you guys.